nestjs-counter/apps/worker/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core';
+import { ExchangeRatesWorkerModule } from './exchange-rates-worker/exchange-rates-worker.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(ExchangeRatesWorkerModule);
+  await app.listen(3001);
```

Comment:
What is this listening for?

Reply:
As far as I can tell, there's no way to start a Nest app without starting an HTTP server. So this just starts an HTTP server that does nothing.

Reply:
I'd add a comment in the code